core/services/keystore/keys/ocr2key/aptos_keyring.go

```diff
@@ -0,0 +1,113 @@
+package ocr2key
+
+import (
+	"crypto/ed25519"
+	"encoding/binary"
+	"io"
+
+	"github.com/hdevalence/ed25519consensus"
+	"github.com/pkg/errors"
```

Comment:
We try to prefer std lib `errors` when possible. Are the files you based this on still using pkg/errors?

Reply:
currently OCR2Keys are using github.com/pkg/errors, so the intention was to follow the same standards.

Should we be using std lib errors instead? if so does the other OCR2Keyrings be changed as well? If so can create another PR for that to reduce changes for this PR

Reply:
currently OCR2Keys are using github.com/pkg/errors, so the intention was to follow the same standards.

Should we be using std lib errors instead? if so does the other OCR2Keyrings be changed as well? If so can create another PR for that to reduce changes for this PR

Reply:
currently OCR2Keys are using github.com/pkg/errors, so the intention was to follow the same standards.

Should we be using std lib errors instead? if so does the other OCR2Keyrings be changed as well? 

I can create another ticket/PR for that to reduce changes for this PR

Reply:
It is fine to match the local style for now. There are some places where we haven't given up pkg/errors because we don't want to give up the stack traces yet. We might have a different solution in the future.